compose.yml

```diff
@@ -191,5 +202,7 @@ services:
             - ./_logs/restic:/var/log
 
 networks:
+    default:
```

Comment:
For what is it added?

Reply:
when i was testing I didn't get output from container, I will clean up.